.travis.yml

```
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - 4
-  - 6
-  - 'stable'
+  - 10
+  - 12
```

for projects that i rarely touch, i've grown pretty fond of just having Travis just run the tests on whatever versions happen to be current at runtime.

```suggestion
  - 'lts/*'
  - 'node'
```
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions

if you prefer to be explicit, no offense taken.

---

Thats what I like to do as well, but I was trying to match the pattern that was present.  Happy to switch to that if @ungoldman is cool with it.

---

fine by me